src/CoreWCFTemplates.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
I understand the need for this while you are developing the templates, but should these be incorporated into the main sln so that as people add support, the templates are also under their noses, so they know to update them too?

Reply:
Running unit test requires the current code to be built and packaged thus i moved them in a separate solution to avoid embarassing users who want to work on CoreWCF runtime code. Please see https://github.com/dotnet/aspnetcore/blob/main/src/ProjectTemplates/README.md#running-projecttemplate-tests , i'm trying to do something similar but without the hive dependency.